helmfile.d/0520.fluentd-elasticsearch-logs.yaml

```diff
@@ -48,8 +48,9 @@ releases:
       open:
         FLUENT_ELASTICSEARCH_HOST: '{{ env "ELASTICSEARCH_HOST" }}'
         FLUENT_ELASTICSEARCH_PORT: '{{ env "ELASTICSEARCH_PORT" }}'
+        FLUENT_ELASTICSEARCH_SCHEME: '{{ env "ELASTICSEARCH_SCHEME" }}'
         FLUENT_UID: "0"
-      secret:
-        FLUENT_ELASTICSEARCH_USER: '{{ env "ELASTICSEARCH_USER" }}'
-        FLUENT_ELASTICSEARCH_PASSWORD: '{{ env "ELASTICSEARCH_PASSWORD" }}'
+#      secret:
```

Comment:
Add a conditional rather than comment out. Customer may be using a managed ES in the future.